plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestPhoenixConnectorTest.java

```diff
@@ -307,6 +317,50 @@ public void testMissingColumnsOnInsert()
         assertQuery("SELECT * FROM test_col_insert", "SELECT 1, 'val1', 'val2'");
     }
 
+    @Override
+    public void testTopNPushdown()
+    {
+        assertThat(query("SELECT orderkey FROM orders ORDER BY orderkey LIMIT 10"))
+                .ordered()
+                .isNotFullyPushedDown(TopNNode.class, ExchangeNode.class, ExchangeNode.class, LimitNode.class);
+    }
+
+    @Test
+    public void testUseSortedProperties()
+    {
+        String tableName = "test_propagate_table_scan_sorting_properties";
+        String createTableSql = format("" +
+                        "CREATE TABLE %s AS " +
+                        "SELECT * FROM tpch.tiny.customer",
+                tableName);
+        assertUpdate(createTableSql, 1500L);
+
+        String expected = "SELECT custkey FROM customer ORDER BY 1 NULLS FIRST LIMIT 100";
```

Comment:
I think we don't need NULLS FIRST here
We needed it in the hive test as the default sort order there is NULLS FIRST

Reply:
Same with Phoenix. Default is NULLS FIRST for ASC and NULLS LAST for DESC